.gitignore

```diff
@@ -0,0 +1,196 @@
+## Images
```

Comment:
This file has a lot of irrelevant file types.   Could you filter it down to only appropriate ones for a python package?

Reply:
There are still a lot of irrelevant types here